src/base_obs.c

```diff
@@ -156,8 +159,9 @@ static void update_obss(obss_t *new_obss)
       }
       base_obss.has_pos = 1;
     } else {
+      // TODO(dsk) check for repair failure
       /* There was an error calculating the position solution. */
-      log_error("Error calculating base station position (%ld)\n", ret);
+      log_warn("Error calculating base station position: (%ld). See solution.c or calc_PVT for info.\n", ret);
```

Comment:
I'd axe this, its the kind of message that confuses our customers a lot.. If you want to make it better you could use the string error.